DriveStation/src/org/team1540/client/ui/WebcamUI.java

```diff
@@ -91,10 +94,11 @@ public WebcamUI() {
 	public WebcamUI(String url, Dimension windowSize) {
 		URL = url;
 		this.windowSize = windowSize;
-		
+
 		initialize();
 	}
-	
+
+	@SuppressWarnings("static-access")
```

Comment:
[NO CHANGE NECESSARY] Generally, just write Class.staticVariable instead of classInstance.staticVariable